### PR TITLE
fix(chat): fix duplicated disabled folder with a dot at the end after import of chat/prompt(Issue #1236)

### PR DESCRIPTION
**Pull request by denys-kolomiitsev**
**Description:**

 Fix duplicated disabled folder with a dot at the end after import of chat/prompt

Issues:

- Issue #1236 

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1242.nightly-test.deltixhub.io
>E2E tests status: failed

File: apps/chat/src/store/import-export/importExport.epics.ts [hunk 1]
```diff
@@ -299,12 +299,15 @@ const importConversationsEpic: AppEpic = (action$) =>
                 entities: conversationsListing,
               });
             });
+          const emptyFolders = folders.filter((folder) =>
```
Comment (IlyaBondar) [inline apps/chat/src/store/import-export/importExport.epics.ts]: Why was it named `empty`? I would name it as `conversationFolders`

File: apps/chat/src/store/prompts/prompts.epics.ts [hunk 2]
```diff
@@ -708,11 +708,15 @@ const importPromptsEpic: AppEpic = (action$) =>
             },
           );
 
+          const emptyFolders = promptsHistory.folders.filter((folder) =>
```
Comment (IlyaBondar) [inline apps/chat/src/store/prompts/prompts.epics.ts]: Why was it named `empty`? I would name it as `promptFolders`

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1242.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

File: apps/chat/src/store/import-export/importExport.epics.ts (hunk 1 above)
Comment (denys-kolomiitsev) [reply apps/chat/src/store/import-export/importExport.epics.ts]: This is name for the folders without conversations inside.

File: apps/chat/src/store/prompts/prompts.epics.ts (hunk 2 above)
Comment (denys-kolomiitsev) [reply apps/chat/src/store/prompts/prompts.epics.ts]: This is name for the folders without prompts inside.

**Pull request merged**